src/features/book/Bookslice.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
- [ ] Let's try to not disable Linter checks on our codes as they help us write better and more professional clean code. Disabling Linter checks defies its purpose. Please fix the error on your next submission. 👍🏽

Reply:
resolved! ✔️